### PR TITLE
feat(editor): implemented block styles

### PR DESCRIPTION
**Pull request by Itrulia**
* Changed output of blockStyle to be the name so that the frontend can target it

### CODE REVIEW

File: apps/editor/src/app/locales/de.json [hunk 1]
```diff
@@ -715,6 +715,7 @@
       "authors": "Autor*innen",
       "blocks": {
         "polls": "Umfrage",
+        "blockStyles": "Block Styling",
```
Comment (antkiewiczk) [inline apps/editor/src/app/locales/de.json:718]: i think some translations are missing here

Comment (antkiewiczk) [reply apps/editor/src/app/locales/de.json:718]: also, no french translations

File: apps/editor/src/app/routes/blockStyles/blockStyleList.tsx [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+  | {type: BlockStyleListActionType.Delete; payload: {id: string}}
+
+const mapBlockStyleToFormValue = (blockStyles: BlockStyle[] | null | undefined) =>
+  blockStyles?.reduce((obj, node) => {
+    obj[node.id] = node
+
+    return obj
+  }, {} as Record<string, BlockStyle>) ?? {}
+
+const blockstyleFormValueReducer = (
```
Comment (antkiewiczk) [inline apps/editor/src/app/routes/blockStyles/blockStyleList.tsx:87]: blockStyle <-- capital S

File: apps/editor/src/app/routes/events/eventForm.tsx [hunk 3]
```diff
@@ -101,7 +101,7 @@ export const EventForm = ({event, onChange, create}: EventFormProps) => {
               <Form.Control
                 name="description"
                 value={event.description || []}
-                onChange={(description: RichTextBlockValue) => onChange({description})}
+                onChange={(description: RichTextBlockValue['richText']) => onChange({description})}
```
Comment (antkiewiczk) [inline apps/editor/src/app/routes/events/eventForm.tsx:104]: 👌

**Review by antkiewiczk — APPROVED** [review]
left some non-blocking comments

File: apps/editor/src/app/locales/de.json (hunk 1 above)
Comment (Itrulia) [reply apps/editor/src/app/locales/de.json:718]: Thats intended, it's like the translation I copied. We should define a workflow for this as it used to be that we just do english and Laurent does the rest.

We have a few that we only have in english.

File: apps/editor/src/app/routes/blockStyles/blockStyleList.tsx (hunk 2 above)
Comment (Itrulia) [reply apps/editor/src/app/routes/blockStyles/blockStyleList.tsx:87]: Will be done in the next PR! Thanks for finding

**Pull request merged**